modules/oxide/src/less/theme/components/collection/collection.less

```diff
@@ -17,18 +17,21 @@
 @collection-item-label-disabled-text-color: fade(@collection-item-label-text-color, 50%);
 @collection-item-caret-disabled-text-color: fade(@collection-item-label-text-color, 50%);
 
-@collection-list-item-active-background-color: @toolbar-button-hover-background-color;
-@collection-list-item-label-active-text-color: contrast(@collection-list-item-active-background-color, @color-black, @color-white);
+@collection-item-hover-background-color: #285ec7;
+@collection-item-hover-text-color: #fff;
```

Comment:
Same here if you look at the other things in the file this needs to use some other type of variable not sure what.